### PR TITLE
Retry bisect

### PR DESCRIPTION
**Pull request by ader1990**
this is a proof of concept to make sure bisect results are correct.

Some bug can appear in just one of N runs, which makes bisect inoperable in the current state, as it will find good commits which in fact are bad.

### CODE REVIEW

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 
-env.LISAV2_REMOTE = "https://github.com/lis/LISAv2.git"
-env.LISAV2_BRANCH = "master"
+env.LISAV2_REMOTE = "https://github.com/ader1990/LISAv2-1.git"
```
Comment (LiliDeng) [inline linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: we can make it as parameter, then can change into any repo we want?

File: linux_pipeline/Jenkinsfile_kernel_bisect_controller [hunk 2]
```diff
@@ -41,6 +48,14 @@ properties ([
             choices: 'Ubuntu_18.04.1\nCentOS_7.5',
             name: 'DISTRO_VERSION',
             description: 'Distro version to be tested.'],
+        [$class: 'StringParameterDefinition',
+            name: 'GOOD_TRIES',
+            defaultValue: '1',
+            description: 'How many times to try to test a good commit. If set to 2, then it will check a commit, if it is a good commit, it will try a second time to see if it is a good commit and return the second run result.'],
```
Comment (LiliDeng) [inline linux_pipeline/Jenkinsfile_kernel_bisect_controller]: run case 2 times, if both pass, it is good commit?

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator (hunk 1 above)
Comment (ader1990) [reply linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: this is not needed, as this is just a work in progress to actually track a very hard to bisect bug.

File: linux_pipeline/Jenkinsfile_kernel_bisect_controller (hunk 2 above)
Comment (ader1990) [reply linux_pipeline/Jenkinsfile_kernel_bisect_controller]: that is the idea, to hunt for the first bad run of lisav2

**Comment by ader1990** [comment]
Closing this PR, as it was just a proof of concept which serverd its purpose.

**Pull request closed without merging**